.github/dependabot.yml

```diff
@@ -0,0 +1,17 @@
+version: 2
+updates:
+  - package-ecosystem: "github-actions"
+    # default location of `.github/workflows`
+    directory: "/"
+    open-pull-requests-limit: 10
+    schedule:
+      interval: "weekly"
+
+  - package-ecosystem: "npm"
+    # location of package manifests
+    directory: "/"
+    open-pull-requests-limit: 10
+    schedule:
+      interval: "daily"
+
+# Built with ❤ by [Pipeline Foundation](https://pipeline.foundation)
```

Comment:
Is this a sneaky self-promotion that shouldn't exist? 👀 
```suggestion
```

Reply:
Hi @sasial-dev,
No, it is not sneaky promotion. PR was delivered with no obligations from non profit initiative as clearly stated.
Isn't this the true value of OSS?